.github/workflows/pr-highlight-changes.yml

```diff
@@ -12,7 +12,7 @@ jobs:
             - name: Install prerequisites
               run: |
                   npm install -g pnpm@^6.24.2
-                  npm -g i @wordpress/env@5.0.0
```

Comment:
Is there a reason that we're not pegging `wp-env` to a specific version?

Reply:
@jonathansadowski I think Christopher indicated he wanted it to be latest. I agree pegging it is a good idea. I'll peg it to 5.1.0 for now.